README.md

```diff
@@ -144,6 +144,7 @@ All internal groups are merged if they are maps, they are overwritten only if an
 | `assignedColors` | Map |  {} (Empty Map) | If a link is in this map, the color found in the SimMechanics file is substituted with the one passed through this map. The color is represented by a 4 element vector of containing numbers from 0 to 1 representing the red, green, blue and alpha component.  |
 | `meshFormat` | String |  `stl_binary` | Format of the meshes exported. Allowed values: `stl_binary`, `stl_ascii`, `step` |
 | `exportMeshes` | Boolean |  True | If false, the meshes will not be exported. |
+| `meshQuality` | Integer |  3 | Quality of the meshes exported. The value is between 1 and 10, where 1 is the lowest quality and 10 is the highest quality. NOTE: this is valid for the stl meshes. |
```

Comment:
Could it make sense to add a reference to the `pfcCoordSysExportInstructions::SetQuality` method to which the parameter is passed and perhaps the docs https://support.ptc.com/help/creo_toolkit/otk_cpp_plus/usascii/index.html#page/creo_toolkit/api/dita/t-pfcModel-CoordSysExportInstructions.html#wwID0EJNT6B ? So the user that are intersted in more details on this parameter can look for more info on their own, without inspecting the code (that is not easy for many users).